packages/plugins/README.md

```diff
@@ -1,157 +0,0 @@
-# ChugSplash Tutorial
```

Comment:
Just FYI, removing this file means there will be no readme on NPM. I'm not sure exactly how, but I'm guessing there's probably a way to share the top level readme with the individual packages.

Reply:
Great, thanks for catching that!